website/server/controllers/api-v3/chat.js

```diff
@@ -145,7 +145,8 @@ api.postChat = {
       throw new NotFound('Your chat privileges have been revoked.');
     }
 
-    if (group.privacy !== 'private' && textContainsBannedWords(req.body.message)) {
+    let tavernNames = ['Tavern', 'HabitRPG'];
+    if (tavernNames.indexOf(group.name) !== -1 && textContainsBannedWords(req.body.message)) {
```

Comment:
there's a TAVERN_ID constant that can be used to recognize the tavern instead of the name